playground/backend/internal/db/mapper/precompiled_object_mapper_test.go

```diff
@@ -40,7 +40,7 @@ func TestPrecompiledObjectMapper_ToObjectInfo(t *testing.T) {
 		Categories:      []string{"MOCK_CAT_1", "MOCK_CAT_2", "MOCK_CAT_3"},
 		PipelineOptions: "MOCK_OPTIONS",
```

Comment:
This will also help because the current value fails to be parsed and so the example is not runnable.
```suggestion
		PipelineOptions: "--name1 value1 --name2 value2",
```

Reply:
This is internal data used only for test, the mock data used for local deployment is loaded form different place. I'm not if such change will be within the scope of this PR